tinymce-per-block/src/blocks/image-block/index.js

```diff
@@ -72,5 +72,13 @@ registerBlock( 'image', {
 			attrs: { /* caption: block.caption ,*/ align: block.align },
 			rawContent
 		};
+	},
+	create: () => {
+		return {
```

Comment:
Something I've toyed in my other recent prototypes. Would it be worthwhile to try to reuse logic from `parse`, and force the implementation to accommodate for an "empty" block to parse? I think there's a clearer need [in my case where I have to ensure a common markup structure](https://github.com/aduth/gutenberg/blob/bd8e2fed3748118e253a13ef5db3b8206ad4eebe/blocks/quote-block/quote-block.js#L13-L32) between creating and preparing an existing node for edit, though I quite like that it avoids breaking the `saved <-> edited` mode duality when adding a third case for creation.